src/main/java/org/jabref/cli/ArgumentProcessor.java

```diff
@@ -289,6 +294,66 @@ public void processArguments() {
         if (!cli.isBlank() && !loaded.isEmpty()) {
             uiCommands.add(new UiCommand.OpenDatabases(loaded));
         }
+
+        if (cli.isCheckConsistency()) {
+            checkConsistency(cliPreferences, entryTypesManager);
+        }
+    }
+
+    private void checkConsistency(CliPreferences cliPreferences,
+                                  BibEntryTypesManager entryTypesManager) {
+        String fileName = cli.getCheckConsistency();
+        Optional<String> outputFormat = Optional.ofNullable(cli.getCheckConsistencyOutputFormat());
+
+        if (fileName == null) {
+            System.out.println(Localization.lang("No file specified for consistency check."));
+            return;
+        }
```

Comment:
Move this after line 305, so that you "exit early" / fail fast.

The "outputformat" is not read in this if - and this not necessary.